src/main/java/org/vaadin/example/bookstore/ui/inventory/InventoryView.java

```diff
@@ -14,9 +14,9 @@
 import com.vaadin.flow.router.OptionalParameter;
 import com.vaadin.flow.router.Route;
 import com.vaadin.flow.router.RouteAlias;
-import com.vaadin.samples.bookstore.backend.DataService;
-import com.vaadin.samples.bookstore.backend.data.Product;
-import com.vaadin.samples.bookstore.ui.MainLayout;
+import org.vaadin.example.bookstore.backend.DataService;
+import org.vaadin.example.bookstore.backend.data.Product;
+import org.vaadin.example.bookstore.ui.MainLayout;
 
 /**
  * A view for performing create-read-update-delete operations on products.
```

Comment:
Is the SampleCrudLogic comment here correct? Didn't we change the class names?

Reply:
Done.